yarn.lock

```diff
@@ -3260,19 +3258,12 @@ __metadata:
   languageName: node
   linkType: hard
 
-"bootstrap@npm:3.4.1, bootstrap@npm:^3.3, bootstrap@npm:^3.4.1, bootstrap@npm:~3.4.1":
-  version: 3.4.1
-  resolution: "bootstrap@npm:3.4.1"
-  checksum: 10/c46285f9d46f6d7a716fa2d294a68b83d8da5f394b2df80e96e0015055e78899312186ac998474568949545e8edcea5c396d500fb9de71c2c81ad5bac1fb56b6
-  languageName: node
-  linkType: hard
-
-"bootstrap@npm:>=2.3.2, bootstrap@npm:>=3":
-  version: 5.3.6
-  resolution: "bootstrap@npm:5.3.6"
+"bootstrap@npm:~5.0.0":
+  version: 5.0.2
+  resolution: "bootstrap@npm:5.0.2"
```

Comment:
This ended up in a downgrade? Can we go to at least 5.3.6? (and by extension popperjs/core ^2.11.8, which also is showing a downgrade?)